util/src/main/i18n/templates/command.properties

```diff
@@ -81,3 +81,5 @@ command.message.blocked = {0} is not accepting messages at this time.
 
 # {0} = map name
 command.maps.hover = Click to view {0}
+
+command.mapdev.velocity = Applying Velocity
```

Comment:
A reason you did not do the text like this; `Applying Velocity ({0}, {1}, {2})` ?

Reply:
I thought it was best to leave some formatting out of the translations, but if you think it would be better I'll switch it to that.

Reply:
I was just wondering. I think both works, but that with the formatting the string makes more sense? It's the norm with the other strings and you would save the string building in the command method. I think both works, just wanted to check you didn't forget the option.

Reply:
I was going to switch it around, but then I took a look and actually I wanted to hide the player name if it’s executed on yourself, and show if the target is another player. Seems easier this way 😄